### DISCUSSION TITLE
Matlab: changing the value of specific intestinal permeability

### DISCUSSION THREAD

**Discussion by PriKalra** [Open-Systems-Pharmacology/Forum#747] [Questions & Problems]
Hi All, 

I want to use `get parameter` and `set parameter `in matlab for changing the value of this part: 

![image](https://user-images.githubusercontent.com/13007976/75258120-4b8f5280-57e6-11ea-8c18-6576fc1884c1.png)
Which parameter will I tweak for this? 
`initStruct=initParameter(initStruct,'Caf_CaCo_2|CAF|Permeability','withWarning');` or
`initStruct=initParameter(initStruct,'Caf_CaCo_2|CAF|Specific intestinal permeability (transcellular)','withWarning');`

**Comment by StephanSchaller** [comment]
The latter, if I understood you correctly.

But in any case I strongly recommend to switch to R as the Matlab Toolbox development has been discontinued.

Best,
Stephan

**Comment by PriKalra** [comment]
:'( 

Years of my hardwork and scripts are gonna go to waste... 

Thankyou

**Comment by StephanSchaller** [comment]
Yes, a good opportunity to improve them all! ;-)

**Comment by msevestre** [comment]
@PriKalra This is not as bad. The interface with the Matlab engine will still work of course. Moving forward, however, new features will only be added to the new R Toolbox. The Matlab Toolbox will be in maintenance mode from now on. Existing scripts and code will still work. 
We encourage you to move to the R Toolbox for new project.

**Comment by PriKalra** [comment]
In that case, is there an equivalent to `Generatecodeforxml` in R too? 
Another question: For sensitivity analysis in R, how do I go about it?

**Comment by Yuri05** [comment]
@PriKalra The new R-toolbox was released with the OSP Suite 9 (it must be installed separately - s. the [documentation](https://docs.open-systems-pharmacology.org/working-with-r/r-introduction) for details)

Regarding your last questions:
>is there an equivalent to **Generatecodeforxml** in R too

Not yet

>For sensitivity analysis in R, how do I go about it?

This is desribed in detail in the documentation: [Sensitivity analysis with the R toolbox](https://docs.open-systems-pharmacology.org/working-with-r/r-introduction#sensitivity-analysis)

As already mentioned before: it is still possible to use the Matlab toolbox with the OSP Suite 9. S. [this WIKI](https://github.com/Open-Systems-Pharmacology/OSPSuite.Documentation/wiki/Using-Matlab-Toolbox-with-PK-Sim-MoBi-Version-9.0-or-later) for details.
